addons/mail/static/src/discuss/core/web/discuss_sidebar_categories.xml

```diff
@@ -13,9 +13,13 @@
     <t t-name="mail.DiscussSidebarCategory">
         <t t-set="hover_class" t-value="'btn p-0 text-start opacity-100-hover opacity-75'"/>
         <div class="o-mail-DiscussSidebarCategory d-flex align-items-center my-1" t-att-class="category.extraClass">
-            <div t-attf-class="d-flex align-items-baseline mx-1 {{ hover_class }}" t-on-click="() => (category.open = !category.open)">
+            <div t-if="store.channels.status === 'fetching'" t-attf-class="d-flex align-items-baseline mx-1 {{ hover_class }}">
+                <i class="o-mail-DiscussSidebarCategory-icon small me-1 fa fa-circle-o-notch fa-spin opacity-50"/>
+                <t t-call="mail.DiscussSidebarCategory.name"/>
+            </div>
+            <div t-else="" t-attf-class="d-flex align-items-baseline mx-1 {{ hover_class }}" t-on-click="() => (category.open = !category.open)">
```

Comment:
A bit unfortunate that `t-attf-class="d-flex align-items-baseline mx-1 {{ hover_class }}"` is duplicated